README.md

```diff
@@ -91,30 +91,27 @@ You may also need to install libcairo
 sudo apt-get install libcairo2 libcairo2-dev
 ```
 
-## Running
+## Input data
 
-Before you run, you'll need some data in the `data` directory.
-For convenience, there's a small test dataset available in the `testdata` directory.
-To use this data just copy over like `cp -r testdata data`.
-(See the data processing section below for more info on building custom data.)
+Olmsted uses [json-schema](https://json-schema.org/) to standardize input data. For a human-readable version of the schema, see [olmstedviz.org/schema.html](http://www.olmstedviz.org/schema.html) or view [schema.html](https://github.com/matsengrp/olmsted/blob/master/schema.html
```

Comment:
Totally up to you, but FWIW, I find it way easier to edit markdown docs when each sentence is on a separate line. Also easier to git/version.

Reply:
More importantly, your second schema link appears to be broken. Also, I don't know if you really need two links here.

Reply:
1. Good idea! 
2. That's because master doesn't have schema.html yet, which will happen once this is merged. 

Re: two links, should we keep having schema.html at olmstedviz.org then or just use the second one?